### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.3][IntelliJ][Wipro]:Focus order is illogical aftert opening the Run on webapp window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1446]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Then hit Tab and observe the focus order.

**Expected Result:**
When "Run on web app window" is opened after hitting 'Tab' keyboard focus should be on the first interactive element in the dialog.

**Actual Result:**
When "Run on web app window" is opened after hitting 'Tab' keyboard focus is on "Use existing web app" radio button in the dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![focus order illogical in run on web app](https://user-images.githubusercontent.com/22956130/35729859-de400846-07c4-11e8-849f-3e6c4c700c2b.png)

**Comment by Eskibear** [comment]
"Use existing web app"  is exactly the first element we added. Above elements are generated by IntelliJ IDEA's template, which we have no access to them in the toolkit.

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. The interactive elements above the "Use Existing Web App" are seldom used. The "Name" is auto-generated and the other 2 checkbox are also not covered in our cases. They are in owned by IntelliJ IDEA itself.
* **Quality of experience**: not impacted. Even if the user really want to change values of those elements, here's the workaround, they can open corresponding pages in "Edit Configurations" dialog. It's also owned by IntelliJ IDEA itself.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749893
